tests/wfm/stitch_test.py

```diff
@@ -121,13 +116,10 @@ def _check_lambda_inside_resolution(lam,
                                     check_value=True):
     dlam = 0.5 * dlam_over_lam * lam
     if event_mode:
-        sum_in_range = sc.bin(data,
-                              edges=[
-                                  sc.array(dims=['wavelength'],
-                                           values=[(lam - dlam).value,
-                                                   (lam + dlam).value],
-                                           unit=lam.unit)
-                              ]).bins.sum().data['wavelength', 0]
+        sum_in_range = data.bin(
+            wavelength=sc.array(dims=['wavelength'],
+                                values=[(lam - dlam).value, (lam + dlam).value],
+                                unit=lam.unit)).hist().data['wavelength', 0]
```

Comment:
Something like
```suggestion
        sum_in_range = data.hist(wavelength=sc.concat([lam - dlam, lam + dlam], 'wavelength']))data['wavelength', 0]
```
unrelated, so ignore this is you like.